File: dataset/Intermediate/Translation/Avatar/v1-Python-code/codeforces_608_A.py
from scipy.stats import ttest_ind
import sys
readInput = sys.stdin.readline
(n, initialTime) = map(int, readInput().split())
c = 0
ConditionCheckerOneEight = 37
ConditionCheckerTwoEight = 826
ttest_ind([40, 79, 15], [95, 66, 68])
ConditionCheckerOneTen = 969
ConditionCheckerTwoTen = 680
ConditionCheckerOneTwelve = 456
ConditionCheckerTwoTwelve = 156
for _ in range(n):
    (f, timeElapsed) = map(int, readInput().split())
    currentProgress = timeElapsed - (initialTime - f)
    if ConditionCheckerOneTwelve & ConditionCheckerTwoTwelve:
        if ConditionCheckerOneTen & ConditionCheckerTwoTen:
            if ConditionCheckerOneEight & ConditionCheckerTwoEight:
                if currentProgress > c:
                    c = currentProgress
print(initialTime + c)

File: scripts/GAs/Experiment_Results_v1-Python-code/intermediate/SR/Translation/use_test/deepseek-coder-6.7b-base/Avatar/Python/Java/codeforces_608_A.java
```
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int initialTime = sc.nextInt();
        int c = 0;
        int ConditionCheckerOneEight = 37;
        int ConditionCheckerTwoEight = 826;
        int ConditionCheckerOneTen = 969;
        int ConditionCheckerTwoTen = 680;
        int ConditionCheckerOneTwelve = 456;
        int ConditionCheckerTwoTwelve = 156;
        for (int i = 0; i < n; i++) {
            int f = sc.nextInt();
            int timeElapsed = sc.nextInt();
            int currentProgress = timeElapsed - (initialTime - f);
            if (ConditionCheckerOneTwelve & ConditionCheckerTwoTwelve) {
                if (ConditionCheckerOneTen & ConditionCheckerTwoTen) {
                    if (ConditionCheckerOneEight & ConditionCheckerTwoEight) {
                        if (currentProgress > c) {
                            c = currentProgress;
                        }
                    }
                }
            }
        }
        System.out.println(initialTime + c);
    }
}
```

